testsuite/forge-cli/src/main.rs

```diff
@@ -51,7 +51,7 @@ struct Args {
 #[structopt(about = "Forge success criteria that includes a bunch of performance metrics")]
 pub struct SuccessCriteriaArgs {
     // general options
-    #[structopt(long, default_value = "5000")]
+    #[structopt(long, default_value = "4000")]
```

Comment:
Let's make it 3500 just to have some buffer and not introduce noisy forge failures?

Reply:
done